src/content/en/fundamentals/primers/service-workers/a-basic-service-worker.markdown

```diff
@@ -0,0 +1,170 @@
+
+{% include shared/takeaway.liquid list=page.key-takeaways.tldr %}
+
+## A client registers a service worker
+
+To use a service worker, a client tells the browser to install it by calling
+`register()`. More than one page can implement `register()`. You can implement
+it in either an inline script or a separate JavaScript file. Only the first a
+browser encounters trigger the download. The most basic register implementation
+looks like this:
+
+{% highlight javascript %}
+// Does the browser support service workers?
+if ('serviceWorker' in navigator) {
+  // Yes
+  navigator.serviceWorker.register('/service-worker.js').then(function() {
+    // Actions not covered by this primer.
+  });
+} else {
+  // No, but this client should work anyway.
+};
+{% endhighlight %}
+
+There's little more that needs to be said about clients. Now let's talk about 
+the service worker itself.
+
+{% include shared/note.liquid list=page.notes.promises %}
+
+## Where can the service worker play?
+
+A client tells the service worker where it can operate. This is called the
+service worker's scope, and it's the lowest level of your website where the
+service worker can be used. Let's look at the registration code again, but with
+some changes to the `register()` parameters.
+
+{% highlight javascript %}
+// Does the browser support service workers?
+if ('serviceWorker' in navigator) {
+  // Yes
+  navigator.serviceWorker.register('/service-worker.js', {scope: '/'})
+    .then(function() {
+    // Actions not covered by this primer.
+  });
+} else {
+  // No, but this client should work anyway.
+}
+{% endhighlight %}
+
+This code is functionally identical to the example given in the last section. 
+Both examples specify that the scope of the service worker is the entire domain. 
+If you want to restrict the service worker to part of a site, simply specify it 
+in the scope. 
+
+For example, say that you have an auction site with different features for
+buyers  and sellers. The buyers section is at `example.com/buyers` and the
+sellers section  is at `example.com/sellers`. A service worker with a scope of
+`/buyers` it can only  serve clients under `example.com/buyers`. It cannot serve
+clients under  `example.com/sellers`. Similarly, a service worker with a scope
+of `/sellers` can  only serve clients under `example.com/sellers`.
+
+{% include shared/note.liquid list=page.notes.https-only %}
+
+## A service worker installs and activates
+
+Let’s look at what happens during the installation of the service worker. To set 
+up a service worker, you need to implement two events:
+
+* `install`
+* `activate`
+
+These events run one time for a service worker in a particular scope, no matter 
+how many clients use it.
+
+### Install
+
+Use the install event to get everything this service worker needs to operate.
+This could include initializing caches,  prefetching data, handling old versions
+of the current service worker, or  handling other service workers in
```

Comment:
I've never had to do anything in an `install` event handler to "handle other service workers in the same scope".

Like the previous comment, I guess there are scenarios in which you *might* need to do something, but bringing that up seems beyond the scope of a primer.